Symfony/CS/Tests/Fixer/Contrib/AlignEqualsFixerTest.php

```diff
@@ -31,6 +31,37 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                $a        = 1;
```

Comment:
`do not touch this` is run for *EVERY* case. See https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/fbca2593178d02d77c06a7351dabb4e32d58590e/Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php#L60